cmd/configurations.go

```diff
@@ -43,13 +55,24 @@ var ConfigurationsCmd = &cobra.Command{
 		kubernetes.CheckForCertExpiry()
 	},
 	Example: `
-# List Kubernetes Dapr configurations
+# List all namespace Dapr configurations in Kubernetes mode
 dapr configurations -k
+
+# List define namespace Dapr configurations in Kubernetes mode
+dapr configurations -k --namespace default
+
+# Print define name Dapr configurations in Kubernetes mode
+dapr configurations -k -n target
```

Comment:
`-n` is no longer namespace shorthand.

Reply:
This example use "-n target" for filter name, "target" is resource name,should i change it?